src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/ScoreVariantAnnotations.java

```diff
@@ -121,34 +121,33 @@
  *
  * <ul>
  *     <li>
- *         Scored VCF file and index. The VCF will not be gzipped if the {@value DO_NOT_GZIP_VCF_OUTPUT_LONG_NAME}
+ *         Scored VCF file and index. The VCF will not be gzipped if the "--do-not-gzip-vcf-output"
```

Comment:
Yes, see #8146 and discussion linked above. I left the `@link` tags since they are more parseable, even when not rendered. Unfortunately, I think removing these tags will make the docs harder to maintain, but it seems like that issue is a wontfix at this point! :(